examples/update-demo/1-run-web-proxy.sh

```diff
@@ -24,6 +24,9 @@ echo
 echo "    http://localhost:8001/static/"
 echo
 
+export KUBE_REPO_ROOT=${KUBE_REPO_ROOT-$(dirname $0)/../..}
+export KUBECTL=${KUBECTL-$KUBE_REPO_ROOT/cluster/kubectl.sh}
```

Comment:
I'm not seeing quotes anywhere still, but this is too minor to pick over. I think some time we should just have someone actually have a space in their k8s repo path and see what breaks.